.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -0,0 +1,75 @@
+name: "Update insecure dependencies"
+
+on:
+  workflow_dispatch: { }
+  schedule:
+    - cron: 0 8 * * *
+jobs:
+  update-insecure-dependencies:
+    strategy:
+      matrix:
+        branch:
+          - "release-2.0"
+          - "release-1.8"
+          - "release-1.7"
+          - "release-1.6"
+          - "release-1.5"
```

Comment:
Should we add `master` ?

Reply:
The thing with `master` is that dependabot works with `master` (and does not work with release branches) so I think there would be conflicts. I guess we could try and disable it for master. I'd start with release branches and then maybe move to master if this action proves stable.

Reply:
I'd add master I don't think the conflict will be that common as the CVE check runs daily when dependabot runs once a week